### ISSUE TITLE
_parsedUrl.url is undefined - version 1.0.0-next.3

### ISSUE THREAD

**Issue by gpfunk** [lukeed/polka#96]
Hi thanks for the great library!

This line is throwing `TypeError: Cannot read property 'substring' of undefined` when running custom middleware.

https://github.com/lukeed/polka/blob/602086f1f4f1a6103e0589b04b69f8b4d934f0fc/packages/polka/index.mjs#L30

I believe its caused by this line

https://github.com/lukeed/polka/blob/602086f1f4f1a6103e0589b04b69f8b4d934f0fc/packages/polka/index.mjs#L36

As far as I can tell, the latest version of @polka/url doesn't set the url on the _parsedUrl. Is this a bug? Or am I missing something?

**Comment by lukeed** [comment]
Hey, thanks!

Do you mind showing me how you're using it? I'm using this version and haven't run into this error, so I'm a bit puzzled haha

Thanks

**Comment by gpfunk** [comment]
I'm just trying to simplify the code a little, I'll throw it up shortly, and it'll hopefully be easier for you to detect what I'm doing wrong haha

I did switch that line `req.url = req._parsedUrl.href;` and it seems to be working fine now

**Comment by gpfunk** [comment]
Okay, heres a simplified version that seems to be throwing this error.

```
const polka = require("polka");

const app = polka();

const mock = (req, res) => {
  res.end();
};

const middleware = (req, res, next) => {
  next();
};

app.use("/one", middleware);
app.use("/one/:id", middleware);
app.get("/one/:id", mock);

app.listen(3000, err => {
  if (err) {
    throw err;
  }
  console.log(`> Running on localhost:3000`);
});
```

**Comment by lukeed** [comment]
You were right, thank you! 🙌

**Comment by lukeed** [comment]
Now included in `polka@1.0.0-next.4` – much appreciated
